rdm/init_files/documents/software_plan.md

```diff
@@ -6,611 +6,1292 @@ title: Software Plan
 
 # Purpose
 
-This document describes a set of activities which will be used during software risk management, development, and maintenance of {{ device.name }}.  It is written primarily for software developers.
+This document for {{device.manufacturer}} plans how the software in {{device.name}} will be designed, developed, and maintained. It identifies the deliverables, tasks, roles and responsibilities involved in these processes.
 
-{{ device.name }} is assigned a Class {{ device.safety_class }} software safety class, which means {% if device.safety_class == "A" %}no injury or damage to health{% elif device.safety_class == "B" %}non-serious injury{% else %}death or serious injury{% endif %} could occur if the software fails [[62304:4.3.a]].  See {{ workflow.risk_management_file }} for details. All of the software items that compose the software system are also presumed to have the same Class {{ device.safety_class }} safety class [[62304:4.3.c 62304:4.3.d 62304:4.3.d 62304:4.3.e 62304:4.3.f 62304:4.3.g]].  The primary purpose of this document is to help developers ensure {{ device.name }} is safe and useful while also allowing developers to be productive.  The secondary purpose is to comply with {{ workflow.version_of_62304 }}.
+{% if device.safety_class == "A" %}
 
-[[:In order to assist auditors and regulators, we have included section references to {{ workflow.version_of_62304 }} as well as occasional comments throughout this document.  These references and comments are always placed inside square brackets, and they are not present in the software-developer version of the document.  Other than these comments, the software-developer version is identical to the auditor version of this document.]]
+All software items in {{device.name}} are assigned a Class A software-safety class, which means no injury or damage to health could occur if the software fails [[62304:4.3.a]]. See {{workflow.risk_management_file}} for details.
+
+{% elif device.safety_class == "B" %}
+
+All software items in {{device.name}} are assigned a Class B software-safety class, which means non-serious injury could occur if the software fails [[62304:4.3.a]]. See {{workflow.risk_management_file}} for details.
+
+{% else %}
+
+All software items in {{device.name}} are assigned a Class C software-safety class, which means death or serious injury could occur if the software fails [[62304:4.3.a]]. See {{workflow.risk_management_file}} for details.
+
+{% endif %}
+
+[[Although it would be possible to assign different safety classifications to different software items, to keep the process simple, we simply assign the same classification to all of them. This classification fulfills 62304:4.3.b, 62304:4.3.c, 62304:4.3.d, 62304:4.3.e, 62304:4.3.f, 62304:4.3.g, and 62304:7.2.2.b]]
```

Comment:
Does this mean all software items are class B?

Reply:
That is how the software plan is currently written, although certainly this could be changed without too much effort! I think the main thing is we'd have to more carefully delineate software items and what safety classification they are so that everyone knows what is class A vs class B.